iodata/formats/mol2.py

```diff
@@ -0,0 +1,152 @@
+                if molecule_found:
+                    lit.back(line)
+                    break
+                title = next(lit).strip()
+            if words[0] == "@<TRIPOS>ATOM":
+                atnums, atcoords, atchgs, attypes = _load_helper_atoms(lit)
+                atcharges = {"mol2": atchgs}
+                atffparams = {"attypes": attypes}
+                result = {
+                    'atcoords': atcoords,
+                    'atnums': atnums,
+                    'atcharges': atcharges,
+                    'atffparams': atffparams
+                }
+                if title is not None:
+                    result['title'] = title
+                molecule_found = True
+    if molecule_found is False:
+        raise lit.error("Molecule could not be read")
+    return result
+
+
+def _load_helper_atoms(lit: LineIterator) -> Tuple[np.ndarray, np.ndarray, np.ndarray, tuple]:
+    """Load element numbers, coordinates and atomic charges."""
+    atnums = []
+    atcoords = []
+    atchgs = []
+    attypes = []
+    for line in lit:
+        words = line.split()
+        if len(words) <= 6:
+            break
+        # Assume that the first character of atom type is the element
+        try:
+            atnums.append(sym2num.get(words[5][0].title()))
+        except ValueError:
+            print(f'Can not convert atom type {words[5][0]} to element')
+        attypes.append(words[5])
+        atcoords.append([float(words[2]), float(words[3]), float(words[4])])
+        if len(words) == 9:
+            atchgs.append(float(words[8]))
+        else:
+            atchg.append(0.0)
+    atnums = np.array(atnums, int)
+    atcoords = np.array(atcoords) * angstrom
+    atchgs = np.array(atchgs)
+    attypes = tuple(attypes)
+    return atnums, atcoords, atchgs, attypes
+
+
+@document_load_many("MOL2", ['atcoords', 'atnums', 'atcharges', 'atffparams'], ['title'])
+def load_many(lit: LineIterator) -> Iterator[dict]:
+    """Do not edit this docstring. It will be overwritten."""
+    # MOL2 files with more molecules are a simple concatenation of individual MOL2 files,'
+    # making it trivial to load many frames.
+    while True:
+        try:
+            yield load_one(lit)
+        except IOError:
+            return
+
+
+@document_dump_one("MOL2", ['atcoords', 'atnums'], ['atcharges', 'atffparams',  'title'])
+def dump_one(f: TextIO, data: IOData):
+    """Do not edit this docstring. It will be overwritten."""
+    # The first six lines are reserved for comments
+    print("# Mol2 file created with Iodata", file=f)
+    print("\n\n\n\n\n", file=f)
+    print("@<TRIPOS>MOLECULE", file=f)
+    print(data.title or 'Created with IOData', file=f)
+    print("@<TRIPOS>ATOM", file=f)
+    for i in range(data.natom):
+        n = num2sym[data.atnums[i]]
+        x, y, z = data.atcoords[i] / angstrom
+        out1 = f'{i:6d} {n:2s} {x:10.4f} {y:10.4f} {z:10.4f} '
+        if not data.atcharges:
```

Comment:
```suggestion
        atcharges = data.atcharges.get('mol2')
        if atcharges is not None:
```

Reply:
Changed  and also adapted the part with the atffparams to have the same style.